### PR TITLE
[Converter] Set caps from caps filter for _NNS_MEDIA_ANY type

### PR DESCRIPTION
**Pull request by gichan-jang**
Set source pad caps form caps filter for _NNS_MEDIA_ANY type.

Example launch-line:
```
...
tensor_filter silent=false framework=tensorflow-lite model=tflite_model/ssd_mobilenet_v2_coco.tflite ! \
tensor_decoder mode=flexbuf ! other/flexbuf ! tensor_converter silent=false ! \
other/tensors, num_tensors=2, framerate=30/1, dimensions=\"4:1:1917:1\,91:1917:1:1\", types=\"float32\,float32\" ! \
tensor_decoder mode=bounding_boxes option1=tflite-ssd option2=tflite_model/coco_labels_list.txt option3=tflite_model/box_priors.txt option4=640:480 option5=300:300 ! \
...
```

Related issue: #3191 

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_converter/tensor_converter.c [hunk 1]
```diff
@@ -1785,9 +1785,24 @@ gst_tensor_converter_parse_caps (GstTensorConverter * self,
...
+          gst_tensors_config_from_structure (&config,
+              gst_caps_get_structure (peer_caps, 0));
+
+        gst_caps_unref (peer_caps);
+      }
+
       if (self->mode == _CONVERTER_MODE_CUSTOM_CODE) {
-        gst_tensors_config_init (&config);
+        if (is_fixed)
+          break;
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_converter/tensor_converter.c:1803]: Q. Possibly moving this to line 1800?

Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:1803]: To register an external converter, moving to line 1800 is impossible in this code status.

**Review by helloahn — APPROVED** [review]
LGTM, `tensor_converter` can handle various cases from now on :)

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
LGTM!

**Pull request merged**